.github/workflows/openastronomy-tox.yml

```
@@ -0,0 +1,21 @@
+name: Run Tests and Post Results
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
```

I think this is same as v1 but good practice to pin hash. Dependabot will take care of updates. (You do have to enable Dependabot manually if not already.)
```suggestion
    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@8c0fde6f7e926df6ed7057255d29afa9c1ad5320  # v1.16.0
```